src/Symfony/Component/Workflow/Definition.php

```diff
@@ -32,7 +42,7 @@ final class Definition
      * @param Transition[]         $transitions
      * @param string|string[]|null $initialPlaces
      */
-    public function __construct(array $places, array $transitions, $initialPlaces = null, MetadataStoreInterface $metadataStore = null)
+    public function __construct(array $places, array $transitions, $initialPlaces = null, MetadataStoreInterface $metadataStore = null, array $dispatchedEvents = null)
```

Comment:
I don't think this should be hold by the definition. It's only related to the Workflow implementation, isn't ?